README.md

```diff
@@ -1,71 +1,31 @@
+# Defang Pulumi Provider
 
-This [Dynamic Resource Provider](https://www.pulumi.com/docs/intro/concepts/resources/dynamic-providers/) allows you to manage your container services on the Defang Opinionated Platform. It was modelled after the [Compose Spec](https://www.compose-spec.io).
+![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/DefangLabs/pulumi-defang?label=Version)
 
-## Install
+The Pulumi provider for [Defang - a radically simpler way to develop, deploy, and debug cloud applications](https://defang.io). The easiest way to deploy your Docker Compose project to the cloud with Pulumi.
 
-The provider is available from NPM Package Registry. See [installation instructions](https://www.npmjs.com/package/@defang-io/pulumi-defang).
-```sh
-npm i @defang-io/pulumi-defang
-```
+## Installation
 
-## Example
+The Defang provider is available as a package in most Pulumi languages:
 
-```ts
-import { DefangService } from "@defang-io/pulumi-defang/lib";
+* JavaScript/TypeScript: [`@defang/pulumi`](https://www.npmjs.com/package/@defang/pulumi)
+* Python: [`pulumi-defang`](https://pypi.org/project/pulumi-defang/)
+* Go: [`github.com/checkly/pulumi-defang/sdk/v1/go/defang`](https://github.com/DefangLabs/pulumi-defang)
 
-const service = new DefangService("defang-demo", {
-  // name: "…",                             // defaults to the Pulumi resource name
-  // image: "nginx:latest",                 // use a pre-built container image, or
-  build: {                                  // build from source
-    context: ".",
-    dockerfile: "Dockerfile.dev",
-  },
-  networks: {
-    public: {
-      //aliases: ["demo.example.com"],
-    },
-  },
-  ports: [{ target: 80, protocol: "http", mode: "ingress" }],
-  // fabricDNS: "…",                        // override the Defang Fabric Controller endpoint
-  // platform: "…",                         // "linux/arm64" | "linux/amd64" | "linux" (default)
-  // environment: {
-  //   RDS_HOST: "rds.endpoint",
-  // },
-  // deploy: {
-  //   replicas: 1,
-  //   resources: {
-  //     reservations: { cpus: 0.5, memory: 512 },
-  //   },
-  // },
-  // healthcheck: {
-  //   test: ["CMD", "curl", "-f", "http://localhost:80/"], // for Alpine-based images use ["CMD", "wget", "-q", "-O-", …]
-  //   interval: 30,
-  //   timeout: 10,
-  //   retries: 3,
-  // },
-});
 
-export const fqdn = service.publicFqdn;     // the final FQDN for your service
-export const natIPs = service.natIPs;       // the public NAT IPs of the service
-export const fabricDNS = service.fabricDNS;
-```
+## Authentication
 
-## Environment Variables
+The Checkly provider must be Authenticated with Defang and with your cloud provider.
+Sign up for [Defang](https://www.defang.io) and follow our [integration guide](https://docs.defang.io/docs/tutorials/pulumi-provider-integration/) to get set up.
 
-* `DEFANG_ACCESS_TOKEN` - your access token; defaults to the token file from the CLI
```

Comment:
I'm guessing some of these are still applicable, to the extent that the CLI knows about them?